### ISSUE TITLE
Add alt text to logo

### ISSUE THREAD

**Issue by kburk1997** [rcos/rcos-handbook#38]
**Description of Issue:**

The logo at the top of the page has no alt text, and thus the text "RCOS: Rensselaer Center For Open Source" does not render on screenreaders.
